cumulusci/cli/tests/test_cci.py

```diff
@@ -159,19 +159,22 @@ def test_handle_sentry_event_no_event(self):
 
         cci.handle_sentry_event(config, True)
 
+    @mock.patch('cumulusci.cli.cci.init_logger')
     @mock.patch('cumulusci.cli.cci.CliConfig')
     @mock.patch('cumulusci.cli.cci.check_latest_version')
-    def test_main(self, check_latest_version, CliConfig):
+    def test_main(self, check_latest_version, CliConfig, init_logger):
         CliConfig.return_value = _marker = object()
 
         ctx, result = run_click_command(cci.main)
 
         check_latest_version.assert_called_once()
+        init_logger.assert_called_once()
```

Comment:
this comment probably would have been better suited during the initial refactor but...

i'd appreciate a change in test name, or perhaps a comment (here or in cli.py), to the effect of "we're testing the stuff that should happen before every command execution" -- theres some magic to understanding how pass_context works and the CliConfig/logger are initialized